src/test/java/com/fasterxml/uuid/UUIDGeneratorTest.java

```diff
@@ -698,6 +746,51 @@ private void checkUUIDArrayForCorrectCreationTimeReorder(UUID[] uuidArray,
         }
     }
 
+    // Modified version for Variant 7 (Unix Epoch timestamps)
+    private void checkUUIDArrayForCorrectCreationTimeEpoch(UUID[] uuidArray,
+            long startTime, long endTime)
+    {
+        // we need to convert from 100-nanosecond units (as used in UUIDs)
+        // to millisecond units as used in UTC based time
+        final long MILLI_CONVERSION_FACTOR = 10000L;
```

Comment:
Actually, no, Variant 7 uses time value in milliseconds, not 100 nsec (unlike variants 1 and 6)